src/EditorFeatures/CSharpTest/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -624,5 +629,36 @@ int M(int i)
     }
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        [WorkItemAttribute(36086, "https://github.com/dotnet/roslyn/issues/36086")]
+        public async Task TestSeverity()
+        {
+            var source =
+@"class Program
+{
+    int M(int i)
+    {
+        [||]switch (i)
+        {
+            case 1:
+                return 4;
+            case 2:
+                return 5;
+            case 3:
+                return 6;
+            default:
+                return 7;
+        }
+    }
+}";
+            var warningOption = new CodeStyleOption<bool>(true, NotificationOption.Warning);
+            var options = Option(CSharpCodeStyleOptions.PreferSwitchExpression, warningOption);
```

Comment:
can just say `CodeStyleOptions.TrueWithSuggestionEnforcement`  then just validate you got a suggestion back.